bom/runtime/pom.xml

```diff
@@ -863,6 +865,39 @@
                 <version>${project.version}</version>
             </dependency>
 
+            <dependency>
+                <groupId>com.datastax.oss</groupId>
+                <artifactId>java-driver-core</artifactId>
+                <version>${datastax.java.driver.version}</version>
+                <exclusions>
+                    <exclusion>
+                        <groupId>org.apache.tinkerpop</groupId>
+                        <artifactId>gremlin-core</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.tinkerpop</groupId>
+                        <artifactId>gremlin-driver</artifactId>
+                    </exclusion>
+                    <exclusion>
+                        <groupId>org.apache.tinkerpop</groupId>
+                        <artifactId>tinkergraph-gremlin</artifactId>
+                    </exclusion>
+                    <exclusion>
```

Comment:
Is this because ASM doesn't work with Graal?  I see that ASM is only used by `com.github.jnr:jnr-ffi:jar:2.1.10:compile` and according to JAVA-2616, jnr-ffi is not working anyways with Graal, so isn't it simpler to exclude `jnr-ffi` ?

Reply:
Nit: jnr-ffi can be made to work with _Graal_, but something else in _Quarkus_ is causing it not to work correctly.

Also note that if we exclude jnr-ffi we have no chance of com.datastax.oss.driver.internal.core.os.Native working correctly so we should explicitly force the Java clock in that case.

Reply:
I will be investigating that more as part of https://datastax-oss.atlassian.net/browse/JAVA-2696. The note about `TIMESTAMP_GENERATOR_FORCE_JAVA_CLOCK` is in comments